### DISCUSSION TITLE
Zksynk

### DISCUSSION THREAD

**Discussion by EmmanuelsegunF** [Cyfrin/foundry-full-course-cu#2783] [Q&A] [answered]
I'm trying to use the zksynk bridge but it keeps loading endlessly.. what should i do?

**Comment by EngrPips** [comment, marked as answer]
Hello @EmmanuelsegunF, clear your browser cache then retry or maybe use another browser.

**Reply by EmmanuelsegunF** [reply]
will do. Thanks

**Reply by EngrPips** [reply]
Anytime, Friend. Keep crushing it.

**Reply by EmmanuelsegunF** [reply]
So sorry to disturb but it's still not working.

**Reply by EmmanuelsegunF** [reply]
i thought it was

**Reply by EngrPips** [reply]
It is fine, It will normally work after I clear my browser cache and delete all the browser data. Did you also try another browser already?

**Reply by EmmanuelsegunF** [reply]
yes. it worked thanks

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
